cmd/boost/wallet_cmd.go

```diff
@@ -132,42 +143,77 @@ var walletList = &cli.Command{
 					}
 				}
 
-				row := map[string]interface{}{
-					"Address": addr,
-					"Balance": types.FIL(a.Balance),
-					"Nonce":   a.Nonce,
+				wallet := map[string]interface{}{
+					addressKey: addr,
+					balanceKey: types.FIL(a.Balance),
+					nounceKey:  a.Nonce,
 				}
 				if addr == def {
-					row["Default"] = "X"
+					wallet[defaultKey] = "X"
 				}
 
 				if cctx.Bool("id") {
 					id, err := api.StateLookupID(ctx, addr, types.EmptyTSK)
 					if err != nil {
-						row["ID"] = "n/a"
+						wallet[idKey] = "n/a"
 					} else {
-						row["ID"] = id
+						wallet[idKey] = id
 					}
 				}
 
 				mbal, err := api.StateMarketBalance(ctx, addr, types.EmptyTSK)
 				if err == nil {
-					row["Market(Avail)"] = types.FIL(types.BigSub(mbal.Escrow, mbal.Locked))
-					row["Market(Locked)"] = types.FIL(mbal.Locked)
+					wallet[marketAvailKey] = types.FIL(types.BigSub(mbal.Escrow, mbal.Locked))
+					wallet[marketLockedKey] = types.FIL(mbal.Locked)
 				}
-
-				tw.Write(row)
+				wallets = append(wallets, wallet)
 			}
 		}
 
 		if !cctx.Bool("addr-only") {
-			return tw.Flush(os.Stdout)
+
+			if cctx.Bool("json") {
+				var lowerCaseWallets []map[string]interface{}
+				for _, wallet := range wallets {
+					lowerCaseWallet := make(map[string]interface{})
+					for k, v := range wallet {
+						lowerCaseWallet[strings.ToLower(k)] = v
+					}
+					lowerCaseWallets = append(lowerCaseWallets, lowerCaseWallet)
+				}
+				return cmd.PrintJson(lowerCaseWallets)
+			} else {
+				tw := tablewriter.New(
```

Comment:
It would have been nice to do that, but instead instanciate an empty tablewriter using something like `tablewrite.New()`, and only within the for loop below, dynamically populate the columns with the keys.
That would have been more maintainable.

Reply:
That would be nice, but the way you've refactored this method makes sense to me 👍